### DISCUSSION TITLE
能否使用react native 写小程序

### DISCUSSION THREAD

**Discussion by liho00** [NervJS/taro#14680] [Q & A]
能否使用react native 写小程序，并加载react native小程序在react native的超级app中呢？

**Comment by tourze** [comment]
如果一定要用react native来写小程序，那么唯一可能就是上react-native-web，然后套个小程序webview做壳了。
如果是想在taro里面使用react-native的约束来写小程序，那么可以尝试在系统启动时就用skyline引擎来写，最终会基本兼容RN。

**Reply by liho00** [reply]
好的，我会尝试第一个建议，第二个感觉有点困难实现。。。
谢谢您的建议，感激不尽！
